.rubocop.yml

```diff
@@ -516,26 +515,97 @@ Performance/StringInclude:
 # Rails cops from rubocop-rails
 # https://github.com/rubocop-hq/rubocop-rails/blob/master/config/default.yml
 # TODO: [@thepracticaldev/oss] enable these Rails cops
+
+Rails/ActiveRecordCallbacksOrder:
+  Description: 'Order callback declarations in the order in which they will be executed.'
```

Comment:
This is the only one I am a little hesitant on. 

I think sometimes it can be helpful and more clear to group `save`, `create`, and `destroy` callbacks together rather then in the order they will be executed. For example, putting `before_destroy` up by `save` calls always throws me off bc each callback is being run under different conditions. Sometimes I think those conditions that cause a callback  to trigger can be more clear to group the callbacks by than the order they run since sometimes they dont run at all.

Reply:
Callbacks are a lot like default scopes, most of them should never be added. For the ones we do add, I prefer the full order, because there are other groups (`_validation`, `_initialize`, `_commit`) and even after all these years I'm still not entirely sure in which order they all run.